cmd/integration_link.go

```diff
@@ -414,9 +416,33 @@ List of available integrations:
 
 			utils.CheckForConsent(c.Context, currentApp, utils.ConsentTypeContainers)
 
-			pullRequestID := c.Args().First()
+			pullRequestID, err := strconv.Atoi(c.Args().First())
+			if err != nil {
+				errorQuit(errgo.Notef(err, "invalid pull request id"))
+			}
+
+			pullRequest, err := integrationlink.PullRequest(c.Context, currentApp, pullRequestID)
+			if err != nil {
+				errorQuit(err)
+			}
+
+			if pullRequest.OpenedFromAForkedRepo {
+				awareOfSecurityRisks := c.Bool("aware-of-security-risks")
+				if !awareOfSecurityRisks {
+					io.Info("\nYou are about to deploy a Review App from a Pull Request opened from a fork.")
```

Comment:
suggestion: Shouldn't we add a message to explain again the risk it implies?

Reply:
This is done in the askForConfirmation[..] function